src/components/AddMemorialForm.vue

```diff
@@ -189,22 +189,46 @@ export default {
 
       }
     },
-    methods: {
+    methods:  {
       handleFileUpload(event) {
         this.memorial.photo_upload = event.target.files[0]
       },
+      test()
+            {
+                this.memorial = {
+                    name: 'kevin smith',
+                    gender: 'male',
+                    photo_upload: 'no',
+                    photo_base64: 'nan',
+                    birth_date: '25-11-1954',
+                    passing_date: '25-11-2020',
+                    location: 'jaipur::rajasthan',
+                    prompt: 'none',
+                    prompt_response: 'none'
+                }
+                let testValue=this.makePostableMemorial();
+            },
+      
+      makePostableMemorial()
+            {
```

Comment:
Please keep consistent formatting with the rest of the file. Curl brace on same line as function def